### ISSUE TITLE
Calling sass functions in Javascript file

### ISSUE THREAD

**Issue by lbekkema** [epegzz/sass-vars-loader#22]
Hi there, 

First of all, thank you for writing this great plugin! 

I'm currently moving all of our variables to a constants file where I want to optimize it as much as possible. I'm currently facing an issue where functions are not being called and just showing up as strings. Below is an example of the code that I am using.

Example:

```javascript
module.exports = {
  colors: {
    red: "#abc",
    blue: "#def",
  }
  styles: {
    border: "1px solid color(red)",
  }
}
```

```sass
@function color($color, $shade:false) {
  @if map-has-key($colors, $color) {
    $color: map-get($colors, $color);
    @if $shade {
      @return rgba($color, $shade);
    } @else {
      @return $color;
    }
  }

  @error "\"#{$color}\" is not a valid color.";
}
```

When calling `border: map-get($styles, border)` the string `border: 1px solid color(red)` is being returned, which makes total sense, so when replacing `color(red)` with `#{color(red)}` the output remains the same.

Do you have any idea on how to fix this? Or is it simply not possible to call sass functions within the `constants.js` file?

Thanks for your help!

**Comment by epegzz** [comment]
I cannot think of a quick solution right now, but I'll think about it over the weekend :)

**Comment by epegzz** [comment]
The problem is, that the `color` function is not defined by the time the vars get evaluated.
I could implement a new option called `sassImports` or `sassIncludes`, that would allow you to include the file in which you define the `color`, function before we define the vars.

**webpack config**
```js
          {
            loader: '@epegzz/sass-vars-loader',
            options: {
              sassImports: [path.resolve(__dirname, 'path/to/color.scss')],
              files: [
                path.resolve(__dirname, 'path/to/styles.js')
              ]
            }
          }
```

**colors.scss**
```sass
@function color($color, $shade:false) {
  @if map-has-key($colors, $color) {
    $color: map-get($colors, $color);
    @if $shade {
      @return rgba($color, $shade);
    } @else {
      @return $color;
    }
  }

  @error "\"#{$color}\" is not a valid color.";
}
```

What do you think? :)

**Comment by epegzz** [comment]
I just released `v3.4.0`

With this version, you can include sass files:

```sass
{
  loader: '@epegzz/sass-vars-loader',
  options: {
    files: [
      path.resolve(__dirname, 'path/to/color.scss'), // <-- include your file containing the sass function
      path.resolve(__dirname, 'path/to/styles.js')
    ]
  }
}
```

Let me know, if you're still having issues :)

**Comment by lbekkema** [comment]
That works fantastic, thanks @epegzz!
